python_tests/artifact/test_backend.py

```diff
@@ -80,3 +88,49 @@ def test_list_trial_artifacts(init_storage_with_artifact_meta: MagicMock) -> Non
         {"artifact_id": "id1", "filename": "bar.txt"},
         {"artifact_id": "id2", "filename": "baz.txt"},
     ]
+
+
+class TestProxyStudyArtifact(TestCase):
```

Comment:
Could you use pytest instead of unittest's class-based style since we now use pytest as a test runner?

Reply:
I see. I have revised the code.